extensions/amp-mustache/0.1/amp-mustache.js

```diff
@@ -76,6 +82,14 @@ export class AmpMustache extends AMP.BaseTemplate {
     root./*OK*/innerHTML = sanitized;
     return this.unwrap(root);
   }
+
+  /** @override */
+  renderHtml(html) {
```

Comment:
This should probably just be called `serializeHtml()` and you can call it from `render()` to de-dupe code.